server/odc-service/src/main/java/com/oceanbase/odc/service/flow/task/BaseODCFlowTaskDelegate.java

```diff
@@ -336,6 +337,11 @@ public boolean cancel(boolean mayInterruptIfRunning) {
 
     private void setDownloadLogUrl(@NonNull Long flowInstanceId) throws IOException, NotFoundException {
         TaskEntity taskEntity = taskService.detail(taskId);
+        TaskType taskType = taskEntity.getTaskType();
```

Comment:
we may use try-catch block to wrap this calling:
```java
File logFile = taskService.getLogFile(taskEntity.getCreatorId(), taskId + "", taskEntity.getTaskType(), OdcTaskLogLevel.ALL);
```

we may upload nothing when this calling throw any exceptions

Reply:
The whole method `setDownloadLogUrl` is in a try-catch block.

Reply:
Yes u are right, done!